docs/contents/components/forms/file-input/accessibility.ja.mdx

```diff
@@ -13,10 +13,20 @@ tab: アクセシビリティ
 <FileInput placeholder="ファイルを選択" aria-label="ファイルアップロード" />
 ```
 
+## キーボード操作
+
+| キー             | 説明                                         | 状態 |
+| ---------------- | -------------------------------------------- | ---- |
+| `Tab`            | 要素をフォーカスします。                     | -    |
+| `Space`, `Enter` | フォーカスしている要素をアクティブにします。 | -    |
```

Comment:
Pressing Space or Enter while FileInput is focused does not seem to activate the element.

It seems that the actual state of the component does not match the documentation.

Reply:
In which environment did you try it? This is a feature that has not yet been released.

Reply:
@hirotomoyamada 

Ah, I see.
So it was not implemented because it was tried in the production environment?
I understand. Thank you very much.

Reply:
@zonoryo17 

So, the branch is oriented to `documantation`.

Reply:
@zonoryo17 

Please approval.